src-java/rule-manager/rule-manager-implementation/src/main/java/org/openkilda/rulemanager/factory/generator/service/lacp/LacpReplyRuleGenerator.java

```diff
@@ -0,0 +1,114 @@
+/* Copyright 2021 Telstra Open Source
+ *
+ *   Licensed under the Apache License, Version 2.0 (the "License");
+ *   you may not use this file except in compliance with the License.
+ *   You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   Unless required by applicable law or agreed to in writing, software
+ *   distributed under the License is distributed on an "AS IS" BASIS,
+ *   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *   See the License for the specific language governing permissions and
+ *   limitations under the License.
+ */
+
+package org.openkilda.rulemanager.factory.generator.service.lacp;
+
+import static org.openkilda.model.MeterId.LACP_REPLY_METER_ID;
+import static org.openkilda.rulemanager.Constants.Priority.LACP_RULE_PRIORITY;
+import static org.openkilda.rulemanager.OfTable.INPUT;
+
+import org.openkilda.model.MacAddress;
+import org.openkilda.model.MeterId;
+import org.openkilda.model.Switch;
+import org.openkilda.model.cookie.CookieBase.CookieType;
+import org.openkilda.model.cookie.PortColourCookie;
+import org.openkilda.rulemanager.Field;
+import org.openkilda.rulemanager.FlowSpeakerData;
+import org.openkilda.rulemanager.Instructions;
+import org.openkilda.rulemanager.OfVersion;
+import org.openkilda.rulemanager.ProtoConstants.EthType;
+import org.openkilda.rulemanager.ProtoConstants.PortNumber;
+import org.openkilda.rulemanager.ProtoConstants.PortNumber.SpecialPortType;
+import org.openkilda.rulemanager.RuleManagerConfig;
+import org.openkilda.rulemanager.SpeakerData;
+import org.openkilda.rulemanager.action.PortOutAction;
+import org.openkilda.rulemanager.factory.generator.service.MeteredServiceRuleGenerator;
+import org.openkilda.rulemanager.match.FieldMatch;
+
+import com.google.common.collect.Sets;
+import lombok.Builder;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.Set;
+
+public class LacpReplyRuleGenerator extends MeteredServiceRuleGenerator {
+
+    private final int inPort;
+    private final boolean switchHasOtherLacpPorts;
```

Comment:
`mustCreateMeter` or `useExisingMeter` sounds better for me. Because only caller/creator have some knowledge about whole switch... `switchHasOtherLacpPorts` good name at that level.